src/OrbitGl/App.cpp

```diff
@@ -1093,6 +1093,8 @@ void OrbitApp::StartCapture() {
       enable_introspection, max_local_marker_depth_per_command_buffer, collect_memory_info,
       memory_sampling_period_ns, std::move(capture_event_processor));
 
+  // TODO (187250643): Refactor this to be more readable and maybe remove parts that are not needed
```

Comment:
```suggestion
  // TODO (b/187250643): Refactor this to be more readable and maybe remove parts that are not needed
```

Reply:
done

Reply:
Interestingly enough - if you do http://b/<number> here CLion will show you the status of the bug in the editor.

Reply:
Yep, `b/` is enough to see the status, hence the suggestion.